crossdock/docker-compose.yml

```diff
@@ -48,6 +48,7 @@ services:
 
     test_driver:
         build: .
+        image: jaegertracing/test_driver:latest
```

Comment:
what does this directive do?  We want to make sure we still build from local source

Reply:
It uses this name when it builds the image. If you do `docker-compose pull` it will pull that image, not build